src/db/models/packages.ts

```diff
@@ -26,9 +26,12 @@ export const getPackagesByTenantId = async (tenantId: string) => {
     try {
         const result = await db.query.packages.findMany({
             where: and(
-                eq(packages.tenantId, tenantId),
-                eq(packages.status, 'delivered')
+                eq(packages.tenantId, tenantId)
             ),
+            orderBy: [
+                sql`CASE WHEN ${packages.status} = 'delivered' THEN 0 ELSE 1 END ASC`,
+                asc(packages.deliveryTime)
+            ],
```

Comment:
praise: I like how you sorted packages by delivery time!

Reply:
issue: If packages.status is NULL, the sort might break. We should ensure the status is valid before sorting

Reply:
The query orders any package with the status that are not 'delivered' after ones that have `delivered` status. So `null` status wouldn't raise an error. I've adjusted so that if there is `null` as status then I word it as unknown for the status.
![image](https://github.com/user-attachments/assets/d4833faf-609b-42ab-a16c-19abef26c825)